runtime/rococo/src/lib.rs

```diff
@@ -94,14 +117,14 @@ include!(concat!(env!("OUT_DIR"), "/wasm_binary.rs"));
 pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("rococo"),
 	impl_name: create_runtime_str!("parity-rococo-v2.0"),
-	authoring_version: 0,
+	authoring_version: 0, // TODO: Params -> authoring_version: 2
 	spec_version: 9260,
 	impl_version: 0,
 	#[cfg(not(feature = "disable-runtime-api"))]
 	apis: RUNTIME_API_VERSIONS,
 	#[cfg(feature = "disable-runtime-api")]
 	apis: sp_version::create_apis_vec![[]],
-	transaction_version: 0,
+	transaction_version: 12,
```

Comment:
This change of `0`to `12` comes just from mirroring Kusama, though, as indeces in `contruct_runtime!` the bump of `transaction_version` is still needed.
I am aware that the whole point of this PR is mirror Kusama as much as we can, but I don't know to which if it this runtime should have a `12` here instead of just a `1`. The runtime is going to work fine, but the message sent to users might differ ?

I would like to read some thoughts on how important it would be following Kusama here too.


+ this change implies `spec_version` bump

Reply:
It is just saying that the pallets order or/and the dispatchables have been change compared to the previous version.
I would say that setting it to `12` makes sense because pallet indexes will be the same as Kusama, . It will let know to any external tool aligned with Kusama-`12` that is also compatible Rococo.

Reply:
this is fine